src/sources/forus-platform/pug/tpl/pages/landing/home-sponsor-noordoostpolder.pug

```diff
@@ -0,0 +1,31 @@
+landing-navbar(template="navbar-noordoostpolder")
```

Comment:
I think these pages are too similar to home-sponsor.pug Why is this needed?

Reply:
I thought there could be more differences at first - on the footer and navbar, but it's using home-sponsor.pug now